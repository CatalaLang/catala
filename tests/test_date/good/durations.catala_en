## Article

```catala
declaration scope A:
  context x content date
  context y content date
  context z content boolean
  context z2 content boolean
  context z3 content duration
  context d content decimal
  context d2 content decimal

scope A:
  definition x equals |2019-01-01|
  definition y equals |2002-09-30|

  definition z equals y +@ (x -@ y) = x
  definition z3 equals x -@ y
  definition z2 equals z3 = 5937 day

  definition d equals z3 /^ z3
  definition d2 equals z3 /^ 2 day
```
