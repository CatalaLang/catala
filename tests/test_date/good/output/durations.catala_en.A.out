[RESULT] Computation successful! Results:
[RESULT] d = 1.
[RESULT] d2 = 2968.5
[RESULT] x = 2019-01-01
[RESULT] y = 2002-09-30
[RESULT] z = true
[RESULT] z2 = true
[RESULT] z3 = 5937 days
